cluster/gce/gci/configure-kubeapiserver.sh

```diff
@@ -76,8 +76,9 @@ function start-kube-apiserver {
   configure-etcd-params params
 
   params+=" --secure-port=443"
-  if [[ "${ENABLE_APISERVER_INSECURE_PORT:-false}" != "true" ]]; then
-    # Default is :8080
+  if [[ "${ENABLE_APISERVER_INSECURE_PORT:-false}" == "true" ]]; then
+    params+=" --insecure-port=8080"
+  else
```

Comment:
Do we need to keep this? The plan is to remove the flag behavior this release.

Reply:
Actually, I think this is unrelated to this PR, and you can remove this file. I'll pick it up in the follow up PR to remove the insecure port.

Reply:
I removed this file from this PR. Would you mind me changing the default value of `--insecure-port` in a separate PR as finishing the remaining work in https://github.com/tallclair/kubernetes/pull/2?

Reply:
If you want to pick up the remaining work, please do!

Reply:
Thanks, I would love to!